layouts/events/single.html

```diff
@@ -40,11 +40,36 @@ <h3>Event Details</h3>
                         <span>{{ .Params.event.cost }}</span>
                     </div>
                     <div class="my-8">
-                        <!-- Calendly link widget begin -->
-                        <link href="https://assets.calendly.com/assets/external/widget.css" rel="stylesheet">
-                        <script src="https://assets.calendly.com/assets/external/widget.js" type="text/javascript"></script>
-                        <a href="#" class="btn bg-orange-500 border-2 border-orange-600 hover:bg-orange-600 hover:border-orange-600" onclick="Calendly.initPopupWidget({url: '{{ .Params.event.calendly_url }}&hide_event_type_details=1'});return false;">Register Now</a>
-                        <!-- Calendly link widget end -->
+                        {{ if .Params.event.calendly_url }}
+
+                            <!-- Calendly link widget begin -->
+                            <link href="https://assets.calendly.com/assets/external/widget.css" rel="stylesheet">
+                            <script src="https://assets.calendly.com/assets/external/widget.js" type="text/javascript"></script>
+                            <a href="#" class="btn bg-orange-500 border-2 border-orange-600 hover:bg-orange-600 hover:border-orange-600" onclick="Calendly.initPopupWidget({url: '{{ .Params.event.calendly_url }}&hide_event_type_details=1'});return false;">Register Now</a>
+                            <!-- Calendly link widget end -->
+
+                        {{ else if .Params.event.eventbrite_url }}
+
+                            <!-- Noscript content for added SEO -->
+                            <noscript><a href="{{ .Params.event.eventbrite_url }}" rel="noopener noreferrer" target="_blank"></noscript>
```

Comment:
Should there be something else in here? I also don't see a closing `a` tag.

Reply:
It is on line 57. The code snippet is directly from Eventbrite so it is a bit wonky